core/src/main/java/org/mifos/mobilewallet/core/data/fineractcn/api/FineractCNApiManager.kt

```diff
@@ -73,4 +79,12 @@ class FineractCNApiManager {
     fun getCustomerApi(): CustomerService {
         return customerApi
     }
+
+    fun getDepositApi(): DepositService {
```

Comment:
@devansh-299 We can directly delegate these functions to their return values, like:
`fun getDepositApi(): DepositService = depositApi `

Nothing technical, just koans ;-)

Reply:
Thanks @s-ayush2903 this surely could have been done. I will update the PR